updatecli/updatecli.d/git-lfs.yml

```
@@ -0,0 +1,53 @@
+---
+title: "Bump Git-LFS version"
+sources:
+  gitLfsVersion:
+    # Version is retrieved from the GitHub source mirror: if there is a release, then you're sure that the artifacts are available
+    kind: githubRelease
+    name: Get the latest Git-LFS version
+    spec:
+      owner: "git-lfs"
+      repository: "git-lfs"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionFilter:
+        kind: regex
```

Same comment here, please be informed of this bug https://github.com/updatecli/updatecli/issues/244
the semver automatically remove the prefix "v" just before fetching changelogs

```suggestion
        kind: semver
```